code/__DEFINES/~skyrat_defines/combat.dm

```diff
@@ -108,73 +108,84 @@
 #define HEADSMASH_BLOCK_ARMOR 20
 #define SUPLEX_TIMER 3 SECONDS
 
-//TODO: Add a grab state check on the do_afters
-/datum/species/proc/try_grab_maneuver(mob/living/carbon/human/user, mob/living/carbon/human/target, modifiers)
-	var/obj/item/bodypart/affecting = target.get_bodypart(ran_zone(user.zone_selected))
+/// Skyrat change - alt-clicking a human as another human while grappling them tightly makes you try for grappling-based maneuvers.
+/mob/living/carbon/human/AltClick(mob/user)
+	if(ishuman(user))
+		var/mob/living/carbon/human/human_user = user
+		if(human_user != src && human_user.combat_mode && human_user.dna.species.try_grab_maneuver(user, src))
+			return
+	. = ..()
```

Comment:
format the code like this so that it returns early.
```suggestion
	if(!ishuman(user))
	        return ..()
	var/mob/living/carbon/human/human_user = user
	if(human_user == src || !human_user.combat_mode || !human_user.dna.species.try_grab_maneuver(user, src))
		return FALSE
	return TRUE

```